h/templates/layouts/group.html.jinja2

```diff
@@ -8,9 +8,7 @@
 
 {% block content %}
   <div class="content paper">
-    <div class="form-container">
-      {% include "h:templates/includes/flash-messages.html.jinja2" %}
-      {{ self.page_content() }}
-    </div>
+    {% include "h:templates/includes/flash-messages.html.jinja2" %}
```

Comment:
I believe only the Join page displays flash messages now, so this should to the join-specific template. Is that correct @seanh?